geomstats/backend/tensorflow/__init__.py

```diff
@@ -40,25 +40,33 @@ def get_mask_i_float(i, n):
     return mask_i_float
 
 
-def get_mask_float(indices, mask_shape):
+def get_mask_float(indices, mask_shape, dtype=float32):
```

Comment:
Ah, good catch! 🎉 

Actually, this could just be called `index_mask` or `indices_mask` or `mask_from_indices` and we have the `dtype` as parameter. Any chance you could update this?

Reply:
Just to be sure I understand correctly:
- You want to rename the function, or one of its parameters ?
- In the first case, there would only be two parameters, `indices` and `dtype` ?